sql/analyzer/stored_procedures.go

```diff
@@ -275,44 +367,48 @@ func applyProcedures(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, se
 			}
 			return n.WithExternalStoredProcedure(procedures[0]), transform.NewTree, nil
 		default:
-			return n, transform.SameTree, nil
+			return n, transform.NewTree, nil
```

Comment:
Should this be `SameTree`?

Reply:
Not quite, in a previous iteration of the code `CALL` would have changed if it made it that far into the function (I had it terminating earlier) so the default case would return the modified node. I've since made some changes, and now I should cache the result from the first `transform.Node` to return in the 2nd.